proxy-node-gateway/src/main/java/uk/gov/ida/notification/exceptions/mappers/SessionAlreadyExistsExceptionMapper.java

```diff
@@ -1,7 +1,6 @@
 package uk.gov.ida.notification.exceptions.mappers;
 
 import uk.gov.ida.notification.exceptions.SessionAlreadyExistsException;
-import uk.gov.ida.notification.exceptions.SessionMissingException;
 import uk.gov.ida.notification.views.ErrorPageView;
```

Comment:
Can this ExceptionMapper extend BaseExceptionMapper?

Reply:
The mappers in proxy-node-gateway will extend another base mapper that will render and post a SAML form, since those handle exception at the point in the journey that interacts with the connector node and the users. The mappers in proxy-node-shared will return errors to the Gateway and we only want them to return JSON 500 responses.